pkgs/tools/networking/iperf/2.nix

```diff
@@ -6,11 +6,11 @@
 
 stdenv.mkDerivation rec {
```

Comment:
would be nice to get rid of `rec` using finalAttrs . I also think the use of pname in src.url is discouraged in nixpkgs (for instance I sometimes change pname when overriding a software, which could break here).